test/integration/setup/adminclient/client.go

```diff
@@ -0,0 +1,305 @@
+}
+
+// svidRPCs verify access to SVID RPCs
+func svidRPCs(ctx context.Context, conn *grpc.ClientConn, td spiffeid.TrustDomain) {
+	log.Println("Verifing SVID RPCs")
+	// Create SVID client
+	client := svid.NewSVIDClient(conn)
+
+	template := &x509.CertificateRequest{URIs: []*url.URL{td.NewID("new_workload").URL()}}
+	csr, err := x509.CreateCertificateRequest(rand.Reader, template, key)
+	if err != nil {
+		log.Fatalf("Fails to create CSR: %v", err)
+	}
+
+	_, err = client.MintX509SVID(ctx, &svid.MintX509SVIDRequest{
+		Csr: csr,
+	})
+	if err != nil {
+		log.Fatalf("Fails to mint X509 SVID: %v", err)
+	}
+
+	_, err = client.MintJWTSVID(ctx, &svid.MintJWTSVIDRequest{
+		Id:       &types.SPIFFEID{TrustDomain: td.String(), Path: "new_workload"},
+		Audience: []string{"myAud"},
+	})
+	if err != nil {
+		log.Fatalf("Fails to mint JWT SVID: %v", err)
+	}
+}
+
+// bundleRPCs verify access to Bundle RPCs
+func bundleRPCs(ctx context.Context, conn *grpc.ClientConn, td spiffeid.TrustDomain) {
+	log.Println("Verifing Bundle RPCs")
+	client := bundle.NewBundleClient(conn)
+
+	blk, _ := pem.Decode([]byte(testBundle))
+	if _, err := client.AppendBundle(ctx, &bundle.AppendBundleRequest{
+		Bundle: &types.Bundle{
+			TrustDomain: td.String(),
+			X509Authorities: []*types.X509Certificate{
+				{
+					Asn1: blk.Bytes,
+				},
+			},
+		},
+	}); err != nil {
+		log.Fatalf("Fails to append bundle: %v", err)
+	}
+
+	_, err := client.BatchCreateFederatedBundle(ctx, &bundle.BatchCreateFederatedBundleRequest{
+		Bundle: []*types.Bundle{
+			{
+				TrustDomain: "foo",
+				X509Authorities: []*types.X509Certificate{
+					{
+						Asn1: blk.Bytes,
+					},
+				},
+			},
+		},
+	})
+	if err != nil {
+		log.Fatalf("Fails to create federated bundle: %v", err)
+	}
+
+	_, err = client.BatchUpdateFederatedBundle(ctx, &bundle.BatchUpdateFederatedBundleRequest{
+		Bundle: []*types.Bundle{
+			{
+				TrustDomain: "foo",
+			},
+		},
+	})
+	if err != nil {
+		log.Fatalf("Fails to update federated bundle: %v", err)
+	}
+
+	_, err = client.BatchSetFederatedBundle(ctx, &bundle.BatchSetFederatedBundleRequest{
+		Bundle: []*types.Bundle{
+			{
+				TrustDomain: "bar",
+				X509Authorities: []*types.X509Certificate{
+					{
+						Asn1: blk.Bytes,
+					},
+				},
+			},
+		},
+	})
+	if err != nil {
+		log.Fatalf("Fails to set federated bundle: %v", err)
+	}
+
+	_, err = client.ListFederatedBundles(ctx, &bundle.ListFederatedBundlesRequest{})
+	if err != nil {
+		log.Fatalf("Fails to list federated bundle: %v", err)
+	}
+
+	_, err = client.GetFederatedBundle(ctx, &bundle.GetFederatedBundleRequest{
+		TrustDomain: "foo",
+	})
+	if err != nil {
+		log.Fatalf("Fails to get federated bundle: %v", err)
+	}
+
+	_, err = client.BatchDeleteFederatedBundle(ctx, &bundle.BatchDeleteFederatedBundleRequest{
+		TrustDomains: []string{"foo"},
+	})
+	if err != nil {
+		log.Fatalf("Fails to delete federated bundle: %v", err)
+	}
+}
+
+// entryRPCs verify access to Entry RPCs
```

Comment:
Not sure if "verify access" is completely accurate.

```suggestion
// entryRPCs verifies that Entry RPCs run successfully
```